.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.ts]
+indent_style = space
+indent_size = 4
```

Comment:
This kind of change should be another PR

Reply:
Yeah sorry for that. I had to add it as otherwise auto-formatting would always start changing the indent.